src/css/modalCard.scss

```diff
@@ -36,6 +37,43 @@
     height: 30rem;
     top: 30%;
   }
+
+  &.measure-widget {
+    top: 25%;
+    left: 75%;
+    height: 7.5rem;
+    width: 17rem;
+
+    @include measure-content-style;
```

Comment:
Not a good fit for this context - can import `measureContent.scss` directly into component